src/widget/woverviewrgb.cpp

```diff
@@ -71,24 +84,36 @@ bool WOverviewRGB::drawNextPixmapPart() {
         unsigned char left = pWaveform->getAll(currentCompletion);
         unsigned char right = pWaveform->getAll(currentCompletion + 1);
 
-        low = pWaveform->getLow(currentCompletion);
-        mid = pWaveform->getMid(currentCompletion);
-        high = pWaveform->getHigh(currentCompletion);
+        low  = static_cast<qreal>(pWaveform->getLow(currentCompletion));
```

Comment:
I think if you declare the variables on the lines of first use, the compiler will be smart about optimizing.  so just

qreal low = ...